src/index.ts

```diff
@@ -355,8 +352,6 @@ export const helpers = {
   parseDimension,
   isDateOrDatetimeField,
   makeFieldProposal,
-  insertTokenAfterArgSeparator,
-  insertTokenAfterLeftParenthesis,
```

Comment:
some auto-completes in odoo are using those helpers -> you need to adapt them and probably export `insertTokenAtArgStartingPosition`